config/environments/development.rb

```diff
@@ -1,6 +1,11 @@
+
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Use the lowest log level to ensure availability of diagnostic information
+  # when problems arise.
+  config.log_level = :debug
```

Comment:
@baob Was it intentional to include this hunk in this PR?

Reply:
I can imagine it being useful, just wondering if there are any downsides to having it as the default (e.g. if it's so verbose that it makes normal log output harder to follow)

Reply:
It was deliberate ... took me ages to discover what exactly it was I needed to do. So unless we can identify a downside, I'm voting for keeping it.

Reply:
OK. Would be helpful to understand the value of debug logging though, so that I and others can also benefit from it:

Do you mean it took you ages to figure out how to increase the log level, or how to trigger a flash message, or something else?

Reply:
Took me ages to figure out the configuration option I needed to enable me to write debugging logging statements in code that actually showed up somewhere readable.